contracts/HenkakuBadge.sol

```diff
@@ -1,33 +1,53 @@
-//SPDX-License-Identifier: Unlicense
+//SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
 import "hardhat/console.sol";
 import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract HenkakuBadge is ERC1155, Ownable {
+    using Counters for Counters.Counter;
+    Counters.Counter private _tokenIds;
 
-contract HenkakuBadge is ERC1155 {
     struct Badge {
         bool mintable;
         bool transerable;
         uint256 amount;
         string tokenURI;
     }
 
-    uint256 tokenId;
-    mapping(uint256 => Badge) bages;
+    mapping(uint256 => Badge) public badges;
 
     constructor() ERC1155("") {}
 
-    function badges() public view returns (Badge[] memory) {}
+    function getBadges() public view returns (Badge[] memory) {}
```

Comment:
This is different from the implementation part, but this function is to get the registered batches, right?
I don't see the difference between badges and getBadges.

Reply:
i made typo and then name conflict occurred https://github.com/henkaku-center/badge/pull/12/files#diff-373d84479a8729ff7599f9bb201aba0cc5a4128efce0c5521f4bb9a5be6851b9R20